### PR TITLE
Minimal correction - define ';' after each SQL statements.

### PR DESCRIPTION
**Pull request by dmitry-lipetsk**
In common case, need use SET TERM. But IBExpert may process this script without it.

### CODE REVIEW

**Comment by dmitry-lipetsk** [comment]
I forgot pointed in commit title, that it for Firebird :)

**Comment by sdanyliv** [comment]
It is really needed or it's just cosmetic changes?

**Comment by dmitry-lipetsk** [comment]
I tried to create database (from this script) script throught IBExpert and found these problems.

At current time, I can't say - these changes usefully or not :)

**Comment by sdanyliv** [comment]
Our tests will create tables. Just try to setup environment.

**Comment by dmitry-lipetsk** [comment]
I know. I say, that this script can't be executes with problem in other tools for FB/IB.

If it is not problem for you - discard my PR :)

**Comment by sdanyliv** [comment]
Changes may be accepted, but currently i have no environment to test that everything is ok.
Will return to this PR later.

**Comment by dmitry-lipetsk** [comment]
When I run corrected script at empty (newly) database, IBExpert returns the next log (of course) with errors:
```
Line	Message
1	. unsuccessful metadata update. DROP PROCEDURE PERSON_SELECTBYKEY failed. Procedure PERSON_SELECTBYKEY not found. 
2	. unsuccessful metadata update. DROP PROCEDURE PERSON_SELECTALL failed. Procedure PERSON_SELECTALL not found. 
3	. unsuccessful metadata update. DROP PROCEDURE PERSON_SELECTBYNAME failed. Procedure PERSON_SELECTBYNAME not found. 
4	. unsuccessful metadata update. DROP PROCEDURE PERSON_INSERT failed. Procedure PERSON_INSERT not found. 
5	. unsuccessful metadata update. DROP PROCEDURE PERSON_INSERT_OUTPUTPARAMETER failed. Procedure PERSON_INSERT_OUTPUTPARAMETER not found. 
6	. unsuccessful metadata update. DROP PROCEDURE PERSON_UPDATE failed. Procedure PERSON_UPDATE not found. 
7	. unsuccessful metadata update. DROP PROCEDURE PERSON_DELETE failed. Procedure PERSON_DELETE not found. 
8	. unsuccessful metadata update. DROP PROCEDURE PATIENT_SELECTALL failed. Procedure PATIENT_SELECTALL not found. 
9	. unsuccessful metadata update. DROP PROCEDURE PATIENT_SELECTBYNAME failed. Procedure PATIENT_SELECTBYNAME not found. 
10	. unsuccessful metadata update. DROP PROCEDURE OUTREFTEST failed. Procedure OUTREFTEST not found. 
11	. unsuccessful metadata update. DROP PROCEDURE OUTREFENUMTEST failed. Procedure OUTREFENUMTEST not found. 
12	. unsuccessful metadata update. DROP PROCEDURE SCALAR_DATAREADER failed. Procedure SCALAR_DATAREADER not found. 
13	. unsuccessful metadata update. DROP PROCEDURE SCALAR_OUTPUTPARAMETER failed. Procedure SCALAR_OUTPUTPARAMETER not found. 
14	. unsuccessful metadata update. DROP PROCEDURE SCALAR_RETURNPARAMETER failed. Procedure SCALAR_RETURNPARAMETER not found. 
16	. unsuccessful metadata update. DROP TABLE PERSONVIEW failed. SQL error code = -607. Invalid command. View PERSONVIEW does not exist. 
18	. unsuccessful metadata update. DROP TRIGGER CREATE_PERSONID failed. Trigger CREATE_PERSONID not found. 
19	. unsuccessful metadata update. DROP TRIGGER CREATE_DATATYPETEST failed. Trigger CREATE_DATATYPETEST not found. 
20	. unsuccessful metadata update. DROP TRIGGER CREATE_BINARYDATAID failed. Trigger CREATE_BINARYDATAID not found. 
22	. unsuccessful metadata update. DROP TABLE DUAL failed. SQL error code = -607. Invalid command. Table DUAL does not exist. 
23	. unsuccessful metadata update. DROP TABLE DATATYPETEST failed. SQL error code = -607. Invalid command. Table DATATYPETEST does not exist. 
24	. unsuccessful metadata update. DROP TABLE DOCTOR failed. SQL error code = -607. Invalid command. Table DOCTOR does not exist. 
25	. unsuccessful metadata update. DROP TABLE PATIENT failed. SQL error code = -607. Invalid command. Table PATIENT does not exist. 
26	. unsuccessful metadata update. DROP TABLE PERSON failed. SQL error code = -607. Invalid command. Table PERSON does not exist. 
28	. unsuccessful metadata update. DROP SEQUENCE DATATYPEID failed. generator DATATYPEID is not defined. 
29	. unsuccessful metadata update. DROP SEQUENCE PERSONID failed. generator PERSONID is not defined. 
31	. unsuccessful metadata update. DROP FUNCTION RTRIM failed. Function RTRIM not found. 
32	. unsuccessful metadata update. DROP FUNCTION LTRIM failed. Function LTRIM not found. 
57	. unsuccessful metadata update. DROP TABLE INHERITANCEPARENT failed. SQL error code = -607. Invalid command. Table INHERITANCEPARENT does not exist. 
68	. unsuccessful metadata update. DROP TABLE INHERITANCECHILD failed. SQL error code = -607. Invalid command. Table INHERITANCECHILD does not exist. 
211	. unsuccessful metadata update. DROP TABLE PARENT failed. SQL error code = -607. Invalid command. Table PARENT does not exist. 
212	. unsuccessful metadata update. DROP TABLE CHILD failed. SQL error code = -607. Invalid command. Table CHILD does not exist. 
213	. unsuccessful metadata update. DROP TABLE GRANDCHILD failed. SQL error code = -607. Invalid command. Table GRANDCHILD does not exist. 
220	. unsuccessful metadata update. DROP TABLE LINQDATATYPES failed. SQL error code = -607. Invalid command. Table LINQDATATYPES does not exist. 
238	. unsuccessful metadata update. DROP SEQUENCE SEQUENCETESTSEQ failed. generator SEQUENCETESTSEQ is not defined. 
243	. unsuccessful metadata update. DROP TABLE SEQUENCETEST failed. SQL error code = -607. Invalid command. Table SEQUENCETEST does not exist. 
253	. unsuccessful metadata update. DROP TRIGGER CREATE_ID failed. Trigger CREATE_ID not found. 
256	. unsuccessful metadata update. DROP SEQUENCE TESTIDENTITYID failed. generator TESTIDENTITYID is not defined. 
259	. unsuccessful metadata update. DROP TABLE TESTIDENTITY failed. SQL error code = -607. Invalid command. Table TESTIDENTITY does not exist. 
279	. unsuccessful metadata update. DROP TRIGGER ALLTYPES_ID failed. Trigger ALLTYPES_ID not found. 
282	. unsuccessful metadata update. DROP SEQUENCE ALLTYPESID failed. generator ALLTYPESID is not defined. 
285	. unsuccessful metadata update. DROP TABLE ALLTYPES failed. SQL error code = -607. Invalid command. Table ALLTYPES does not exist. 
699	. unsuccessful metadata update. DROP TABLE CamelCaseName failed. SQL error code = -607. Invalid command. Table CamelCaseName does not exist. 
714	. unsuccessful metadata update. DROP TABLE TESTMERGE1 failed. SQL error code = -607. Invalid command. Table TESTMERGE1 does not exist. 
715	. unsuccessful metadata update. DROP TABLE TESTMERGE2 failed. SQL error code = -607. Invalid command. Table TESTMERGE2 does not exist. 
```
Repeated execution returns the error:
```
Line	Message
20	. unsuccessful metadata update. DROP TRIGGER CREATE_BINARYDATAID failed. Trigger CREATE_BINARYDATAID not found. 
```

**Comment by sdanyliv** [comment]
My point that current unit test have not to fail :)
I have never deal with IB and it's tools,

File: Source/LinqToDB/Data/DataConnection.cs [hunk 1]
```diff
@@ -854,6 +854,8 @@ public IDbConnection Connection
 		{
 			get
 			{
+				this.ThrowOnDisposed();
```
Comment (sdanyliv) [inline Source/LinqToDB/Data/DataConnection.cs]: Why this "fix" is here?

**Comment by sdanyliv** [comment]
@dmitry-lipetsk. For each PR create branch in your fork. Do not use your master!

File: Source/LinqToDB/Data/DataConnection.cs (hunk 1 above)
Comment (dmitry-lipetsk) [reply Source/LinqToDB/Data/DataConnection.cs]: Sorry, I not have enought expirence with git :)

**Comment by sdanyliv** [comment]
Fixed in 0e7c861d6d4fbf7230e9f40fda7627827e49f019

**Pull request closed without merging**